src/AutoMerge/toml.jl

```diff
@@ -0,0 +1,18 @@
+function maybe_parse_toml(directory::AbstractString, file::AbstractString)
```

Comment:
I would just write this as

```julia
maybe_parse_toml(f) = isfile(f) ? TOML.parsefile(f) : Dict{String, Any}()
```

All these error checking etc seems like it should be up to the caller. And most functions take a full path to a file, not a `directory` + `file`.

Reply:
Done:
1. `maybe_parse_toml` takes the full path to a file.
2. `maybe_parse_toml` is defined in one line, using a ternary.
3. The error checking is done by each caller.